packages/client/hmi-client/src/components/drilldown/tera-drilldown-header.vue

```diff
@@ -3,7 +3,9 @@
 		<div class="title-row">
 			<h4><slot /> <i v-if="props.tooltip" v-tooltip="tooltip" class="pi pi-info-circle" /></h4>
 			<slot name="inputs" />
-			<!-- <a :href="documentationUrl" rel="noopener noreferrer">Documentation</a> -->
+			<a v-if="documentationUrl" :href="documentationUrl" target="_blank" rel="noopener noreferrer"
```

Comment:
Can you remove the target blank.
Unless specifically asked by Pascale for demo reasons links should open normally.

Reply:
Done